poofarmer/Scripts/EnemySpawner.gd

```diff
@@ -52,10 +63,13 @@ func _on_SpawnTimer_timeout():
 		enemy.fleeingVector = spawnLoc
 		enemy.speed *= 1 + (wave_count - 1) / 10
 		enemy.connect("global_poo_stolen", player, "_on_Goblin_global_poo_stolen")
+		enemy.connect("enemy_killed", goblins, "_on_enemy_killed")
 		
 		if population_countdown > 0:
 			population_countdown -= 1
 			goblins.add_child(enemy)
```

Comment:
I feel like the trolls should spawn into a Trolls node not the goblin node

Reply:
I just didn't bother to rename that node to `Enemies`. I can go through and find everywhere that node and variable is referenced and rename it if you want. I don't think there should be a separate node. That would make things way too complicated with the enemy count and the gobodar.

Reply:
My point is that it may be better to have the different mob types in their own nodes, and then we add a helper method to the spawner that adds the child_counts together.

Reply:
Okay I see how that could be helpful. I'll make that change